scripts/gha/trigger_workflow.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Trigger a GitHub workflow_dispatch workflow using the GitHub REST API.
+
+This script allows you to easily trigger a workflow on GitHub using an access
+token. It uses the GitHub REST API documented here:
+https://docs.github.com/en/rest/reference/actions#create-a-workflow-dispatch-event
+
+Usage:
+python trigger_workflow.py -w workflow_filename -t github_token [-b branch_name]
+       [-r git_repo_url] [-p <input1> <value1> -p <input2> <value2> ...]'
+       [-C curl_command]
+
+If -r is unspecified, uses the current repo.
+If -c is unspecified, uses the current HEAD.
+"""
+
```

Comment:
Same comments for this file. Especially if we have more files that we already made in CPP, it's worth to checkout the cpp repo in the workflow and use the same script in these two repos.